backend/open_webui/routers/orq.py

```
@@ -0,0 +1,169 @@
+log = logging.getLogger(__name__)
+log.setLevel(SRC_LOG_LEVELS.get("ORQ", logging.INFO))
+
+router = APIRouter()
+
+# Configuration Models
+class ORQConfigForm(BaseModel):
+    ENABLE_ORQ_API: Optional[bool] = None
+    ORQ_API_KEY: str
+
+@router.get("/config")
+async def get_config(request: Request, user=Depends(get_admin_user)):
```

Ruff (0.12.2)

Unused function argument: `user`

(ARG001)